src/modm/platform/uart/stm32/module.lb

```diff
@@ -84,6 +84,14 @@ def prepare(module, options):
             or device.has_driver("lpuart:stm32*")):
         return False
 
+    module.add_option(
+        BooleanOption(
+            name="with_freertos",
+            description="Enable FreeRTOS buffers for UART",
+            default=False,
+            dependencies=lambda v: {False: None,
+                                    True: ":freertos"}[v]))
+
```

Comment:
I'd do it the other way around. Move the `freertos.hpp` file to `ext/aws/` and let the `:freertos` module copy it to `modm/platform/uart/` folder. That way it's implicit and you don't need another option.

Reply:
Agree, that looks much better. Done.